charts/demo-smtp/templates/deployment.yaml

```diff
@@ -22,6 +22,9 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          env:
+            - name: RELAY_NETWORKS
+              value: "{{ .Values.config.relayNetworks }}"
```

Comment:
Can you make this optional? (currently, if left unspecified, the chart does not deploy)

Reply:
Ah yes, meant to do so, thank u!

Reply:
I'd actually rather fix it like this https://github.com/wireapp/wire-server-deploy/pull/11/commits/9803a9f1a5af1af58e663526e058ecb2aa00a13f